pyproject.toml

```diff
@@ -13,6 +13,8 @@ omit = [
     # These are covered by `tests/integration/test_runner_manager_openstack.py`.
     "src/openstack_cloud/openstack_cloud.py",
     "src/openstack_cloud/openstack_runner_manager.py",
+    # Thin wrapper around GitHub API. Not a lot of value in unit tests.
+    "src/manager/github_runner_manager.py",
```

Comment:
Minor: Perhaps it would be better to put a `# pragma: no cover` in the classes of the source file, so that we are reminded every time we modify the module about the exclusion. Because the implementation may get more complex in the future, and we may want to have coverage, and by excluding the whole file we'll never get notified about missing coverage.

Reply:
Good idea, I will update the code.